hs/src/Reach/Eval/Module.hs

```diff
@@ -41,7 +41,7 @@ evalImportClause env im =
       (at', ns) <- withAt $ flip parseIdent ji
       return $ M.singleton ns $ (SLSSVal at' Public $ SLV_Object at' (Just $ "module " <> ns) env)
     JSImportClauseNamed (JSImportsNamed _ iscl _) ->
-      evalImExportSpecifiers (LC_RefFrom "module import") env go iscl
+      evalImExportSpecifiers LC_RefFrom env go iscl
```

Comment:
I don't understand why this is removed... because we do want to continue to get this nice error... but maybe I'll see something good when I get to the examples